CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2220,6 +2220,150 @@ private void OnMouseButton(MouseButtonEventArgs e)
             }
         }
 
+        /// <summary>
+        /// Provides class handling for the <see cref="E:System.Windows.TouchDown" /> routed event that occurs when a touch presses inside this element.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.TouchEventArgs" /> that contains the event data.
+        protected override void OnTouchDown(TouchEventArgs e)
+        {
+            Focus();
+            CaptureTouch(e.TouchDevice);
+            OnTouch(e);
+            base.OnTouchDown(e);
+        }
+
+        /// <summary>
+        /// Provides class handling for the <see cref="E:System.Windows.TouchMove" /> routed event that occurs when a touch moves while inside this element.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.TouchEventArgs" /> that contains the event data.
+        protected override void OnTouchMove(TouchEventArgs e)
+        {
+            OnTouch(e);
+            base.OnTouchMove(e);
+        }
+
+        /// <summary>
+        /// Provides class handling for the <see cref="E:System.Windows.TouchUp" /> routed event that occurs when a touch is released inside this element.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.TouchEventArgs" /> that contains the event data.
+        protected override void OnTouchUp(TouchEventArgs e)
+        {
+            ReleaseTouchCapture(e.TouchDevice);
+            OnTouch(e);
+            base.OnTouchUp(e);
+        }
+
+        /// <summary>
+        /// Handles a <see cref="E:Touch" /> event.
+        /// </summary>
+        /// <param name="e">The <see cref="TouchEventArgs"/> instance containing the event data.</param>
+        private void OnTouch(TouchEventArgs e)
+        {
+            var modifiers = e.GetModifiers();
+            var touchPoint = e.GetTouchPoint(this);
+            var touchEventType = TouchEventType.Cancelled;
+            switch (touchPoint.Action)
+            {
+                case TouchAction.Down:
+                    touchEventType = TouchEventType.Pressed;
+                    break;
+                case TouchAction.Move:
+                    touchEventType = TouchEventType.Moved;
+                    break;
+                case TouchAction.Up:
+                    touchEventType = TouchEventType.Released;
+                    break;
+                default:
+                    touchEventType = TouchEventType.Cancelled;
+                    break;
+            }
+
+            var touchEvent = new TouchEvent()
+            {
+                Id = e.TouchDevice.Id,
+                X = (float)touchPoint.Position.X,
+                Y = (float)touchPoint.Position.Y,
+                RadiusX = 0,
```

Comment:
Shouldn't this default to `0` already? (And the others like it).

Reply:
Removed unnecessary initializers.